fms/models/gpt_bigcode.py

```diff
@@ -0,0 +1,320 @@
+import math
+from dataclasses import dataclass
+from typing import Optional, Tuple
+
+import torch
+import torch.nn as nn
+from fms.utils.config import ModelConfig
+from fms.modules.feedforward import FeedForwardBlock
+from fms.utils.activation import str_to_activation
+
+from fms.modules.attention import MultiHeadAttention
+
+
+@dataclass
+class GPTBigCodeConfig(ModelConfig):
+    src_vocab_size: int = 49157  # original is 49152, this is based on len(tokenizer)
+    emb_dim: int = 2048
```

Comment:
same q here on source of configs... HF model is 768. is this just due to multiple variants/sizes?

Reply:
ya I mean it depends on which variation of the model you are using. I based this number on https://huggingface.co/bigcode/gpt_bigcode-santacoder

Reply:
let's add a comment referring to this default variant/source. it's not the top google search so it might be confusing otherwise to understand which configuration we start with.

Reply:
Updated in latest commit